### PR TITLE
Added Laser Musket and BB gun

### PR DESCRIPTION
**Pull request by nokele**
Added Laser Musket craftable from a MF Cell, Nuka Cola bottle, pipe rifle and electronic parts.
Very energy inefficient (4 shots total), double the damage of an AE9
Added BB gun with 2000 damage, does not exist anywhere yet, someone change damage and such.

### CODE REVIEW

File: code/modules/projectiles/projectile.dm [hunk 1]
```diff
@@ -792,6 +792,11 @@
...
 		M.IgniteMob()
 
+//other
 
 /obj/item/projectile/bullet/F13/c38mmBullet
 	damage = 4
+
+
+/obj/item/projectile/bullet/F13/bbBullet
+	damage = 1000
```
[the remark this answers is not in the document]
Comment (Laus4Deus) [reply code/modules/projectiles/projectile.dm:802]: (So we can have it in game and not just as a meme thing that will never see the daylight)
(In fact, it having 100 bb's is pretty good; you can fire without needing to reload and whatnot)
(I am thinking h*ck just make it semi-auto, but that would force you to move the entire damn thing to another parent)

File: code/modules/crafting/recipes.dm [hunk 2]
```diff
@@ -1734,8 +1734,9 @@
 	reqs = list(/obj/item/stock_parts/cell/ammo/mfc = 1,
 				/obj/item/stack/crafting/electronicparts = 2,
 				 /obj/item/gun/ballistic/revolver/pipe_rifle = 1,
-				 /obj/item/reagent_containers/food/drinks/bottle/nukacola = 1)
-	tools = list(TOOL_WORKBENCH)
+				 /obj/item/reagent_containers/food/drinks/bottle/nukacola = 1,
+				 /obj/item/stack/cable_coil)
```
Comment (Laus4Deus) [inline code/modules/crafting/recipes.dm:1738]: 😖 `/obj/item/stack/cable_coil = 3)` please

**Comment by nokele** [comment]
updated

**Comment by judgex** [comment]
Please resolve this conflict and let me know when it's ready.

**Comment by nokele** [comment]
finally, had to merge some stuff from the new update

**Pull request closed without merging**